specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -2618,6 +2663,54 @@
         }
       }
     },
+    "CheckEligibilityDefinition": {
+      "type": "object",
+      "description": "The check eligibility Connector credentials definition",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Connector kind (eg aws)"
+        },
+        "properties": {
```

Comment:
Why wrap everything in `properties` here? You're not defining a resource, just a POST API.

Reply:
will fix